pkg/agent/flowexporter/connections/connections_test.go

```diff
@@ -286,5 +297,16 @@ func TestConnectionStore_DeleteConnectionByKey(t *testing.T) {
 		assert.Nil(t, err, "DeleteConnectionByKey should return nil")
 		_, exists := connStore.GetConnByKey(*testFlowKeys[i])
 		assert.Equal(t, exists, false, "connection should be deleted in connection store")
+		checkAntreaConnectionMetrics(t, len(connStore.connections))
 	}
 }
+
+func checkAntreaConnectionMetrics(t *testing.T, numConns int) {
+	expectedAntreaConnectionCount := `
+	# HELP antrea_agent_conntrack_antrea_connection_count [ALPHA] Number of connections in the Antrea ZoneID of the conntrack table.
+	# TYPE antrea_agent_conntrack_antrea_connection_count gauge
+	`
+	expectedAntreaConnectionCount = expectedAntreaConnectionCount + fmt.Sprintf("antrea_agent_conntrack_antrea_connection_count %d\n", numConns)
+
+	assert.Equal(t, nil, testutil.GatherAndCompare(legacyregistry.DefaultGatherer, strings.NewReader(expectedAntreaConnectionCount), "antrea_agent_conntrack_antrea_connection_count"))
```

Comment:
`assert.NoError`

Reply:
Thanks, I also fixed it for other 2 asser.Equal() call.